recipes/travo/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set name = "travo" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}-split
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/travo-{{ version }}.tar.gz
+  sha256: 61da7a967051f93707ba4fe7c13e5399567e9e1eae72ac766dce49e678490f36
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - travo = travo.console_scripts:travo
+    - travo-echo-travo-token = travo.console_scripts:travo_echo_travo_token
+
+outputs:
+  - name: travo
```

I think you need to re-iterate this for all outputs
```suggestion
  - name: travo
    build:
      noarch: python
```